kubernetes/helm_charts/core/analytics/templates/serviceMonitor.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.serviceMonitor.enabled }}
+{{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) ( .Values.serviceMonitor.enabled ) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "analytics.fullname" . }}
+  namespace: {{ default .Values.namespace .Release.Namespace  }}
+  labels:
+    {{- toYaml .Values.serviceMonitor.labels | nindent 4 }}
+spec:
+  endpoints:
+  - honorLabels: true
+    interval: 15s
+    path: /metrics/druid
+    port: {{ .Values.network.port }}
+    scheme: http
+    scrapeTimeout: 10s
+  jobLabel: analytics
+  namespaceSelector:
+    matchNames:
+    - {{ default .Values.namespace .Release.Namespace  }}
+  selector:
+    matchLabels:
+    {{- include "analytics.labels" . | nindent 6 }}
```

Comment:
This won't work as there are no helper functions. Please use the same notation as the deployment file.